### PR TITLE
Document operators on the Operators page

### PR DESCRIPTION
**Pull request by glv**
Currently, the [Operators page](https://hexdocs.pm/elixir/1.10.2/operators.html) includes a precedence table, plus information about comparison operators and customization/overriding. It really should have documentation for *all* of the operators, or at least pointers to that documentation (especially since the [Syntax reference page](https://hexdocs.pm/elixir/1.10.2/syntax-reference.html#operators) points you there for a "full reference"). This change fixes that situation.

I'm an Elixir newbie, so I may be wrong about some of the details here. Please point out anything I got wrong!

(Also: I noticed this when I was trying to learn the actual semantics and allowable usage of `in` and `not in`, and it took me a long time to finally find their documentation under Kernel / Guards. So I also added short notes to `Enum.member?/2` and `Enumerable.member?/2` for other newcomers like me.)

### CODE REVIEW

File: lib/elixir/pages/Operators.md [hunk 1]
```diff
@@ -29,6 +29,43 @@ Operator
...
+Additionally, there are a few other operators that Elixir parses but doesn't actually use.
+See [Custom and overridden operators](#custom-and-overridden-operators) below for a list and for guidelines about their use.
+
+Some other operators are special forms and cannot be overridden:
+
+  * [`^`](`^/1`) - pin
+  * [`.`](`./2`) - remote and anonymous calls, and alias join
+  * [`=`](`=/2`) - match
+  * [`&`](`&/1`) - function capture
+  * [`::`](Kernel.SpecialForms.html#::/2) - type
```
Comment (glv) [inline lib/elixir/pages/Operators.md]: This didn't generate the right link when I just had `::/2` in the parens.

File: lib/elixir/pages/Operators.md [hunk 2]
```diff
@@ -29,9 +29,46 @@ Operator
...
+
+  * `=>` - see [`%{}`](`%{}/1`)
+  * `when` - see [Guards](patterns-and-guards.html#guards)
+  * `<-` - see [`for`](`for/1`) and [`with`](`with/1`)
+  * `\\` - see [Default arguments](Kernel.html#def/2-default-arguments)
+
 ## Comparison operators
 
-Elixir provides the following built-in comparison operators:
+Elixir provides the following built-in comparison operators (all of which can be overridden or used in guards):
```
Comment (josevalim) [inline lib/elixir/pages/Operators.md]:
```suggestion
Elixir provides the following built-in comparison operators (all of which can be used in guards):
```

While it is true they can be overridden, it is extremely discouraged, so I don't want to "promote" it too much. :)

File: lib/elixir/pages/Operators.md (hunk 1 above)
Comment (josevalim) [reply lib/elixir/pages/Operators.md]: Sounds like a bug in `ExDoc`. We will investigate, thanks!

**Comment by josevalim** [comment]
Hi @glv! This is absolutely great, thank you! :green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by glv** [comment]
@josevalim You're welcome! Glad to finally be working with your wonderful language!

**Pull request merged**